File: src/main/java/pl/mjedynak/idea/plugins/builder/action/GenerateBuilderAction.java
package pl.mjedynak.idea.plugins.builder.action;

import com.intellij.openapi.editor.actionSystem.EditorAction;
import pl.mjedynak.idea.plugins.builder.action.handler.GenerateBuilderActionHandler;

public class GenerateBuilderAction extends EditorAction {
    public GenerateBuilderAction() {
        super(new GenerateBuilderActionHandler());
    }
}

File: src/main/kotlin/io/github/kingg22/kobra/builder/action/GenerateBuilderAction.kt
package io.github.kingg22.kobra.builder.action

import com.intellij.openapi.editor.actionSystem.EditorAction
import io.github.kingg22.kobra.builder.action.handler.GenerateBuilderActionHandler

public class GenerateBuilderAction : EditorAction(GenerateBuilderActionHandler())
